### DISCUSSION TITLE
How to disable spell checking?

### DISCUSSION THREAD

**Discussion by thegrasshopper104** [suchnsuch/Tangent#80] [Q&A] [answered]
I can't find a setting to disable spell checking or any info about in the documentation. It's pretty annoying when typing not in English.
Can you disable it?

**Comment by taylorhadden** [comment, marked as answer]
Hi there! You should be able to disable spell checking in the "Notes" section of the settings panel. On Windows, you can _also_ add additional language dictionaries for spell check, which may solve your problem better!

![image](https://github.com/suchnsuch/Tangent/assets/15878743/e193f845-eaa7-421c-8f4e-a04d706f7927)

**Reply by thegrasshopper104** [reply]
Thanks. It's just there I just missed it.
